README.md

```diff
@@ -1,634 +1,1855 @@
+support a healthy under-collateralized loan market. **You do not need DIDs.**
+
+> If people want to take back control from governments, they need to accept more responsibility.
+> *Freedom isn't free.*  
+> -- quote from previous section
+
+### Multi-Asset Loans
+
+While this protocol technically only allows one asset to be the loan asset for each loan, the fact
+that this protocol is composable with itself means you can effectively create "packaged" loans where
+each loan is for a different loan asset. For example, imagine if Alice wants to borrower 2000 ADA
+worth of assets, but is concerned about the price volatility of ADA. Instead of doing a single loan
+for 2000 ADA, she can do 1 loan in ADA, 1 loan in DJED, and 1 loan in another asset; the total value
+borrowed would be equivalent to 2000 ADA. Since she can ask for them in one transaction, accept them
+in one transaction, and make payments on all of them in one transaction; practically speaking, they
+are effectively a single loan with three loan assets. By enabling borrowers to create these packaged
+loans, borrowers can hedge themselves against "global" price movements of any one asset.
+
+### Staking Script Support
+
+Supporting staking script credentials allows using any kind of custom logic to protect identities.
+One major use case for this is enabling corporations to adopt DeFi. Most corporations will not be
+comfortable having their identities only protected by a single pubkey. At the very least, these
+corporations will prefer using multisig native scripts. Furthermore, there will likely be intense
+pressure from regulators for corporations to use at least a multisig for all DeFi activities.
+
+### Endogenous Price and Interest Rate Discovery
+
+Cardano-Loans is designed to be independent from the traditional financial system, in favor of
+endogenously producing its own. As such, every piece of a loan, including the relative values of
+assets to collateral, interest rates, and term length are all negotiated and agreed upon fully p2p -
+no oracle feeds required. 
+
+Although this presents a bootstrapping problem, it may be overcome by the fact that this protocol
+may be the fastest way for the most financially underserved peoples to begin building a credit
+history. Prospective borrowers are incentivized to build p2p relationships within a global
+marketplace of prospective lenders, who themselves are incentivized to lend by the relatively high
+rates that financially underserved borrowers would be willing to pay. Since offers are public,
+lenders can see all offers being made to the borrower and/or for that loan asset, resulting in a
+healthy competition between lenders, and a healthy interest-rate discovery mechanism. And since this
+protocol does not rely on any team or "utility token", this market is immune to any top-down
+influence.
+
+With enough users and liquidity, this protocol may eventually *serve* as the de-facto oracle for
+market-driven interest-rate discovery.
```

Comment:
Love it.  Lofty goals, but possible.  I would definitely love to see that eventuate.